dashboard/config/custom_session_store/instrumented_cookie_store.rb

```diff
@@ -0,0 +1,45 @@
+require 'action_dispatch/middleware/session/cookie_store'
+require 'objspace'
+
+module ActionDispatch
+  module Session
+    # Simple wrapper around the standard Rails decentralized cookie-based
+    # session store that we currently use in production. The primary goal is to
+    # collect some data about current usage patterns, to inform our decisions
+    # about a possible switch to a centralized session store.
+    #
+    # TODO infra: remove this helper after we've gathered enough data.
```

Comment:
I'd lowkey prefer to see this TODO link back to a "remove this code" PR, even if its an unstarted PR for now. You're probably more rememberful than I am, but I discovered a few of these in my career that I'd forgotten when something urgent came up, and one of them was a regretful waste of resources for a couple years.

Reply:
Great idea! It is far too easy to forget about TODOs :upside_down_face: